src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -255,11 +255,21 @@ class GitOpsConfiguration extends Component<GitOpsProps, GitOpsState> {
             return;
         }
         this.setState({ validationStatus: VALIDATION_STATUS.LOADER, saveLoading: true });
+
+        let finalHostUrl
```

Comment:
let trimmedHostUrlStr = this.state.form.host;
if (trimmedHostUrlStr) {
   try{
let trimmedHostUrl = new URL(trimmedHostUrlStr);
trimmedHostUrlStr = trimmedHostUrl.protocol + "//" + trimmedHostUrl.host + "/"
}catch(e){
}
}


host : trimmedHostUrlStr